bungeecord/src/main/java/io/github/retrooper/packetevents/bungee/factory/BungeePacketEventsBuilder.java

```diff
@@ -99,7 +99,7 @@ public ServerVersion getVersion() {
                         String bungeeVersion =
                                 ProxyServer.getInstance().getVersion();
                         for (final ServerVersion val : ServerVersion.reversedValues()) {
-                            if (bungeeVersion.contains(val.getReleaseName())) {
+                        	if(val.getReleaseName().contains(bungeeVersion)) {
```

Comment:
```suggestion
                        	if (val.getReleaseName().contains(bungeeVersion)) {
```

Reply:
Not sure why VSCode sorted the imports and applied Prettier to everything but this one if. Resolved.